cryptographic_estimators/UOVEstimator/uov_problem.py

```diff
@@ -0,0 +1,87 @@
+# ****************************************************************************
+# Copyright 2023 Technology Innovation Institute
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+# ****************************************************************************
+
+
+from ..base_problem import BaseProblem
+from sage.arith.misc import is_prime_power
+
+
+class UOVProblem(BaseProblem):
+    """
+    Construct an instance of UOVProblem.
+
+    INPUT: 
+        
+        - ``n`` -- number of variables
+        - ``m`` -- number of polynomials
+        - ``q`` -- order of the finite field (default: None)
```

Comment:
q must be given to create a UOV instance it cannot be  neither None nor having any default value